packages/schemas/src/CoreSchemaWithIcons.ts

```diff
@@ -52,4 +53,8 @@ export const SCHEMA_TYPES_WITH_ICONS: Record<
 		description: "A set of 3 numbers",
 		icon: BoxSelect,
 	},
+	[CoreSchemaType.Null]: {
+		description: "An empty value",
+		icon: CircleSlash,
+	},
```

Comment:
Only the CircleSlash renders anywhere right now. FYI @ktyjscott I randomly chose this one, but happy to replace if you find one that is more suitable!